src/lib/components/SearchBar/overriden/ResponsiveSearchBar.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Input } from 'semantic-ui-react';
+import { screenIsWiderThan } from '@components/utils';
+import { Breakpoints } from '@components/Media';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.updateQueryString = this.props.updateQueryString;
+    this.state = {
+      currentValue: this.props.queryString || '',
+    };
+  }
+
+  onInputChange = (queryString) => {
+    this.setState({
+      currentValue: queryString,
+    });
+  };
+
+  executeSearch = () => {
+    this.updateQueryString(this.state.currentValue);
+  };
+
+  onBtnSearchClick = (event, input) => {
+    this.executeSearch();
+  };
+
+  onKeyPress = (event, input) => {
+    if (event.key === 'Enter') {
+      this.executeSearch();
+    }
+  };
+
+  render() {
+    const {
+      actionProps,
+      autofocus,
+      responsiveAutofocus,
+      executeSearch,
+      onBtnSearchClick,
+      onInputChange,
+      onKeyPress,
+      overridableId,
+      placeholder,
+      uiProps,
+    } = this.props;
+    let shouldFocus = autofocus;
+    if (responsiveAutofocus) {
```

Comment:
Note for reviewers: This is the same searchBar from RSK, the only difference is the new `responsiveAutofocus` prop.